packages/mdc-slider/_mixins.scss

```diff
@@ -435,8 +435,16 @@
   $feat-color: mdc-feature-create-target($query, color);
 
   .mdc-slider__track-container {
-    @include mdc-feature-targets($feat-color) {
-      @include mdc-theme-prop(background-color, rgba(mdc-theme-prop-value($color), $opacity));
+    &::after {
```

Comment:
How is this using pseudo selector different from directly applying on the element?

Reply:
This comment on the issue over in material-components-web-components has more context:  https://github.com/material-components/material-components-web-components/issues/80#issuecomment-535065517

Without the pseudo selector, the color of the slider track defaults to the primary color because it uses the mdc-theme-prop-value which cannot handle secondary css variable definitions.  Additionally, because rgba is used here previously, css variables will not function properly because passing in a css variable that is a hex value to an rgba function will invalid.  The only way to make this work with css variables is by using a pseudo-class that fills the parent element and applying that css variable to the background-color and setting an opacity value.  Applying an opacity value directly to the track container will also not work because that will then also apply to its children which means the track itself will be opaque which is not the desired state either.